flows/runs/run.go

```diff
@@ -296,6 +298,27 @@ func (r *flowRun) Context(env envs.Environment) map[string]types.XValue {
 	}
 }
 
+// returns the context representation of the current node
+//
+//   uuid:text -> the UUID of the node
+//   visit_count:number -> the count of visits to the node by this contact
+//
+// @context node
+func (r *flowRun) nodeContext(env envs.Environment) map[string]types.XValue {
+	_, node, _ := r.PathLocation()
+	visitCount := 0
+	for _, s := range r.path {
+		if s.NodeUUID() == node.UUID() {
+			visitCount++
+		}
+	}
```

Comment:
Kinda feels to me like this should only be calculated when evaluated, iterating the entire path comparing UUIDs isn't the end of the world but then again it isn't nothing either.

Reply:
Most of the context is lazy loaded already using `flows.Context` or `flows.ContextFunc` - in this case it's loaded at the point of `@node` being accessed in the root.

Reply:
Ah, ok, perfect.